src/client/Microsoft.Identity.Client/TelemetryCore/Internal/Events/ApiEvent.cs

```diff
@@ -169,6 +170,12 @@ public int RegionSource
             set => this[RegionSourceKey] = (value).ToString(CultureInfo.InvariantCulture);
         }
 
+        public string ValidateUseRegion
```

Comment:
Hmm maybe rename to something like `IsValidUserProvidedRegion` and maybe make it a nullable bool?

Reply:
Good point. Updated.